packages/shared/src/components/onboarding/OnboardingPlusStep.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { FC, useState } from 'react';
+import classNames from 'classnames';
+import { useViewSize, ViewSize } from '../../hooks';
+import {
+  ProductOption,
+  usePaymentContext,
+} from '../../contexts/PaymentContext';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../typography/Typography';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { PlusComparingCards } from '../plus/PlusComparingCards';
+
+interface OnboardingStepProps {
+  onClickNext: () => void;
+}
+
+const PlusBillingCycleSwitch: FC<{
```

Comment:
Looks to be something we can reuse in the future. Might be good to take it to its own file.

Reply:
Why FC though 😢